commandlineparser/include/Options.h

```diff
@@ -1,29 +1,32 @@
-#include<iostream>
-#include<string>
+#include <iostream>
+#include <string>
+#ifndef OPTIONS_H
```

Comment:
Guards needs to be at the top of the file

Reply:
done .

Reply:
Where is the commit for this ?